ci/docker-compose.yml

```diff
@@ -56,7 +56,7 @@ services:
     volumes:
       - authn-local:/run/authn-local
       # TODO: authenticators_oidc/test has a dep on this
-      - ./test_suites/authenticators_oidc/keycloak:/authn-oidc/keycloak/scripts
+      - ./oauth/keycloak:/authn-oidc/keycloak/scripts
```

Comment:
do we want the scripts not to sit in `authn-oidc` also inside the container? will these scripts be used outside of the oidc tests?

Reply:
i will change it also to: `/oauth/keycloak/scripts` , thanks